lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss

```diff
@@ -176,13 +180,32 @@
   }
 
   .button--file {
+    font-weight: $weight-bold;
     height: auto;
+    margin: ($base-spacing / 2) 0 0;
+    padding: 0;
+    text-transform: lowercase;
     width: auto;
   }
 }
 
+.reportback__submissions-latest {
+  background-color: #fff;  // remove
```

Comment:
Just FYI, was purposely left here. Next few PRs will deal with items within this element and will be addressed then.